service/build.gradle

```diff
@@ -42,14 +42,7 @@ dependencies {
 	implementation group: 'bio.terra', name: 'terra-cloud-resource-lib', version: "1.2.7-SNAPSHOT"
 
 	// Sam
-	implementation group: "org.broadinstitute.dsde.workbench", name: "sam-client_2.13", version: "0.1-fd8ee25"
-
-	// Transitive dependency constraints due to security vulnerabilities in prior versions.
-	// These are not directly included, they are just constrained if they are pulled in as
-	// transitive dependencies.
-	constraints {
-		implementation('org.json:json:20231013')
-	}
+	implementation group: "org.broadinstitute.dsde.workbench", name: "sam-client_2.13", version: "0.1-0c4b377"
```

Comment:
Where'd this version come from?  @tlangs told me! https://broadinstitute.slack.com/archives/C03GMG4DUSE/p1707940443578919?thread_ts=1707857380.637899&cid=C03GMG4DUSE

Reply:
And, we can look ahead to a happy future where the Sam client is published with semver: https://broadworkbench.atlassian.net/browse/ID-1091